src/Illuminate/Database/Connectors/SqlServerConnector.php

```diff
@@ -94,6 +94,10 @@ protected function getSqlSrvDsn(array $config)
             $arguments['ApplicationIntent'] = 'ReadOnly';
         }
 
+        if (isset($config['connectionPooling']) && $config['connectionPooling'] == false) {
```

Comment:
we don't want to keep mixing up camelCase and snake_case in our config files - avoiding it is best so nobody has to go look it up in order to use the config